Makefile.build

```diff
@@ -723,6 +723,10 @@ plugins/micromega/%.cmo: plugins/micromega/%.ml
 	$(SHOW)'OCAMLC    $<'
 	$(HIDE)$(OCAMLC) $(COND_BYTEFLAGS) -c $<
 
+tools/coq_makefile.cmo: tools/coq_makefile.ml tools/CoqMakefile.in
+	$(SHOW)'OCAMLC    $<'
+	$(HIDE)$(OCAMLC) $(COND_BYTEFLAGS) -c $<
```

Comment:
You do not need to put an actual rule. You can just write a single line

    tools/coq_makefile.cmo: tools/CoqMakefile.in

to mark that `tools/CoqMakefile.in` is an extra dependency of `tools/coq_makefile.cmo`.

But that does not even make sense to me in the first place, since the content of `tools/coq_makefile.cmo` does not depend on the content of `tools/CoqMakefile.in`, does it? (Same thing for `tools/coq_makefile.cmx`.)

Reply:
Thanks, learning something new every day. (I could have deduced this as this is how in coq_makefile, the dependencies for .v files are added in addition to a matching pattern rule)

Yes, you are correct, this is not needed. (I was convinced that locally, I changed the Makefile.in, yet the behaviour of the old version persisted, but after studying the setup again in the light of a fresh day, I'm sure that it was my fault, I most likely did forget to save before running the tests again.)